Distiller-server/src/main/java/com/mapr/distiller/server/Coordinator.java

```diff
@@ -2115,19 +2116,70 @@ public void run() {
 							+ " as it was disabled after retrieval from schedule.");
 				}
 			}
-
-			/*
-			 * if (System.currentTimeMillis() >= lastStatus + statusInterval) {
-			 * lastStatus = System.currentTimeMillis();
-			 * LOG.debug("Main: Printing status " + lastStatus); String
-			 * recordProducerStatus = getRecordProducerStatus();
-			 * 
-			 * LOG.debug("Record Queues"); String recordQueues =
-			 * getRecordQueues();
-			 * 
-			 * LOG.debug("Metric Actions"); String metricActions =
-			 * getMetricActions(); }
-			 */
+			/**
+			 * Some status stuff that is useful in debugging
+			if(System.currentTimeMillis() >= lastStatus + statusInterval){
```

Comment:
I have rolled all these statuses into appropriate methods, we can add methods for logPersistorStatus as well. I know you are having this just for testing purposes. But should this need to be checked in?